content/docs/user-guide/project-config/_index.md

```diff
@@ -32,16 +33,49 @@ To change any of the properties in this file, you can either edit the file manua
 
-| Property | Description | Default |
-| --- | --- | --- |
-| **project_name** | The name of the project. The `--project-name` O3DE CLI parameter uses this name to identify the project. | "<PROJECT_NAME>" |
-| **origin** | A URL for the project, such as the repository URL or project website. This field is intended for Project Manager to use. | "The primary repo for <PROJECT_NAME> goes here: i.e. http://www.mydomain.com" |
-| **license** | The license that the project uses, and any copyright information that you want to include. Consider providing the URL for the license. This field is intended for Project Manager to use. | "What license <PROJECT_NAME> uses goes here: i.e. https://opensource.org/licenses/MIT" |
-| **display_name** | The display name of the project in Project Manager. | "<PROJECT_NAME>" |
-| **summary** | A short description of the project. This field is intended for Project Manager to use. | "A short description of <PROJECT_NAME>." |
-| **canonical_tags** | A standard field in O3DE manifests used to identify an O3DE object's type. Examples: "Gem", "Project". Projects should use the "Project" tag. | [ "Project" ] |
-| **user_tags** | Any keyword tags that you want to associate with the project. Examples: "Physics", "Assets", "AWS". For examples of the standard set of tags that Gems use, refer to the Project Manager **Gem Catalog**. Project Manager uses these tags for documentation, search, and filtering. | [ "<PROJECT_NAME>" ] |
-| **icon_path** | The path and filename of the project icon. Project Manager uses this icon. The file must be located in the project root directory, and the filename must be `preview.png`. <br><br> WARNING: Do not change the path or filename. | "preview.png" |
-| **engine** | The name of the engine registered to the project. Register engines in your O3DE manifest, located at `<USER>/.o3de/o3de_manifest.json`. | "o3de" |
-| **restricted_name** | (Optional) The name of a restricted folder to associate with the project. | "projects" |
-| **external_subdirectories** | (Optional) The path to one or more directories to include in the project build. You can use any directory with a `CMakeLists.txt` file. When you register Gems to the project using the `register --external-subdirectory-project-path` O3DE CLI command, O3DE adds their paths here. | |
+| Property | Required | Description | Default |
+| --- | --- | --- | --- |
+| **project_name** | **Required** | The name of the project. The `--project-name` O3DE CLI parameter uses this name to identify the project. | "<PROJECT_NAME>" |
+| **display_name** | **Required** | The display name of the project in Project Manager. | "<PROJECT_NAME>" |
+| **engine** | Optional | The name of the engine and optional version specifier this project uses. Register engines in your O3DE manifest, located at `<USER>/.o3de/o3de_manifest.json`. | "o3de" |
```

Comment:
We should provide a preamble as well before this table to indicate that <USER> stands for the User Home Directory.

Such as 
```
The <USER> placeholder below refers to the user's home directory based on the operating being used
|OS|Home|
|---|---|
|Windows|`C:\Users\<YourUserName>`|
|Linux|`/home/<YourUserName>`|
|MacOS|`/Users/<YourUserName>`|

For example if your user name is "Foo" and you are working on Windows machine, the <USER> directory would be `C:\Users\Foo`
```